web/packages/api/src/assets_v2.ts

```diff
@@ -750,6 +856,7 @@ export function padFeeByPercentage(fee: bigint, padPercent: bigint) {
     if (padPercent < 0 || padPercent > 100) {
         throw Error(`padPercent ${padPercent} not in range of 0 to 100.`)
     }
+    console.log("PAD: ", fee * ((100n + padPercent) / 100n))
```

Comment:
Remove this logging?

Reply:
Removed in https://github.com/Snowfork/snowbridge/pull/1463/commits/90a7ddca2b9b88ec1a5c6b0001c5e774b478b3a1.